<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Abstract_Transactions</class>
 <widget class="QMainWindow" name="Abstract_Transactions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 127);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>581</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 254, 255);</string>
    </property>
    <property name="title">
     <string>Transaction page</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Please Enter your card number :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_part1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_part2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_part3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_part4">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_confirm">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 248, 255);
font: 600 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Timer:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_second">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_minute">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionBack"/>
   </widget>
   <addaction name="menuProfile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
